cli/compose/loader/testdata/v1-full.yml

```diff
@@ -0,0 +1,10 @@
+net:
+  image: busybox
+volume:
+  image: busybox
+  volumes:
```

Comment:
if this is a "full" test it should probably include all the fields supported by v1 and one service with build, one with image

Reply:
@dnephin yeah I just copied the file from `docker/compose so far :stuck_out_tongue: